CHANGELOG.md

```diff
@@ -4,10 +4,15 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Bump to k8s 1.24 and kube 0.73.1 ([#408]).
```

Comment:
I think this is breaking. At least the reconcile method parameter changed in [0.73.0](https://github.com/kube-rs/kube-rs/releases/tag/0.73.0)
```
-async fn reconcile(doc: Arc<MyObject>, context: Context<Data>) -> Result<Action, Error>
+async fn reconcile(doc: Arc<MyObject>, context: Arc<Data>) -> Result<Action, Error>
```

Reply:
Yup, upgrading a reexported crate to a new semver-major version (or semver-minor for 0.x) is always breaking.

Reply:
marked as breaking